projects/rsv/index.js

```diff
@@ -0,0 +1,38 @@
+const sdk = require("@defillama/sdk");
+const abi = require("./abi.json");
+const { sumTokensAndLPsSharedOwners } = require("../helper/unwrapLPs");
+
+const vaultContract = "0xAeDCFcdD80573c2a312d15d6Bb9d921a01E4FB0f";
```

Comment:
is this reserve? I think it's already integrated

Reply:
Idk, but this protocol was on github as requested one!